cylc/flow/scripts/remove.py

```diff
@@ -18,7 +18,27 @@
 
 """cylc remove [OPTIONS] ARGS
 
-Remove one or more task instances from a running workflow.
+Remove tasks from a running workflow and the workflow's history.
+
+By default, the specified task(s) will be removed from all flows, making it
+as if the task(s) never ran. The task(s) and their outputs will be left in the
+`None` flow to preserve a record that they ran, however.
+
+If you specify flow(s) to remove a task from (not all flows that it belongs to)
+then the task will still exist in the remaining flow(s), but it will not
+influence the evolution of the workflow in the specified flow(s).
+
+Removing a submitted or running task will also kill it (see "cylc kill").
```

Comment:
This mixes up the default and the specific a bit: you can remove a task from all flows explicitly as well as by default. How about this:

```suggestion
By default, the specified task(s) will be removed from all flows.

Tasks removed from all flows, and any downstream tasks spawned by
their outputs, will be recorded in the `none` flow and will not affect
the evolution of the workflow.

If you remove a task from some but not all of its flows, it will still exist
in the remaining flows, but it will not affect the evolution of the
workflow in the removed flows.

Removing a submitted or running task will also kill it (see "cylc kill").
```